ArduCopter/standby.cpp

```diff
@@ -21,3 +21,25 @@ void Copter::standby_update()
     attitude_control->set_yaw_target_to_current_heading();
     pos_control->standby_xyz_reset();
 }
+
+// enter standby mode, flight controller is no longer flying the vehicle
+void Copter::enter_standby()
+{
+    standby_active = true;
+    AP::logger().Write_Event(LogEvent::STANDBY_ENABLE);
+    gcs().send_text(MAV_SEVERITY_INFO, "Stand By Enabled");
```

Comment:
nitpick: I think we should change this to "StandBy" instead of "Stand By".  It's existing code so not a blocker of course.

Reply:
Yeh, I agree. My bad.

Reply:
fixed